### PR TITLE
fix: fix readme and support version command

### PR DESCRIPTION
**Pull request by flywukong**
### Description

fix the description of cmd about the version

### Rationale

help user use the correct version

### Example

NA

### Changes

Notable changes:
* fix the description of cmd about the version

### CODE REVIEW

**Review by sysvm — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -11,14 +11,16 @@ for any bug bounty. We advise you to be careful and experiment on the network at
 
 ## Cmd usage
 
-The latest release version is compatible with the greenfield. If the user need to use the command tool on the test net of greenfield, he will need to switch to the latest release version.
+Please notice that the latest release version is compatible with the greenfield system of test net.
```
Comment (KeefeL) [inline README.md:14]: Greenfield is still undergoing rapid development iterations, and greenfield-cmd also needs to be continuously updated and adapted. When using it, please do not directly use the master branch or develop branch. If you are using this tool on the Greenfield Testnet, please switch to the latest official release version. To obtain the latest version of the command tool, please visit the following URL: https://github.com/bnb-chain/greenfield-cmd/releases.

File: README.md [hunk 2]
```diff
@@ -11,14 +11,16 @@ for any bug bounty. We advise you to be careful and experiment on the network at
...
+If the user need to use the command tool on the test net of greenfield, he will need to switch to the latest release version.
+To obtain the latest version of the command tool, please visit the following URL: https://github.com/bnb-chain/greenfield-cmd/releases.
+
 ### installation
 
 ```
 git clone https://github.com/bnb-chain/greenfield-cmd.git
 cd greenfield-cmd
-# To obtain the latest version of the command tool, please visit the following URL: https://github.com/bnb-chain/greenfield-cmd/releases.
-git checkout -b v0.0.8
```
Comment (KeefeL) [inline README.md:21]: Add a line: # Find the latest release here: https://github.com/bnb-chain/greenfield-cmd/releases

File: README.md [hunk 3]
```diff
@@ -37,7 +39,7 @@ The rpcAddr indicates the Tendermint RPC address with the port info.
 rpcAddr = "https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443"
 chainId = "greenfield_5600-1"
 ```
-The command tool can also support other networks besides the test network. 
+The command tool can also support other networks besides the test net.
```
Comment (KeefeL) [inline README.md:42]: test net -> Testnet

File: README.md (hunk 1 above)
Comment (flywukong) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (flywukong) [reply README.md:21]: added

File: README.md (hunk 3 above)
Comment (flywukong) [reply README.md]: fixed

**Pull request merged**